model/dao/ProjectDAO/PostgreSQLProjectDAO.php

```diff
@@ -462,8 +462,6 @@ public function getByCustomerUserLogin($customerId = NULL, $userLogin = NULL,
      * @param DateTime $filterEndDate end date of the time filter for projects.
      *        Projects will a start date sooner than this date will be returned.
      *        NULL to deactivate filtering by this field.
-     * @param boolean $activation filter projects by their activation field.
-     *        NULL to deactivate filtering by this field.
```

Comment:
The removal of the redundant `$activation` parameter should have belonged to a different commit. Remember, different fixes belong to different commits :)